js/src/rive_advanced.mjs.d.ts

```diff
@@ -300,26 +576,48 @@ interface RiveOptions {
     get bottomCenter(): Alignment;
     get bottomRight(): Alignment;
   }
-  
+
+  /**
+   * Axis-aligned bounding box
+   */
   export interface AABB {
     minX: number;
     minY: number;
     maxX: number;
     maxY: number;
   }
-  
+
+  /**
+   * Matrix described by the following:
```

Comment:
Can we call it a "Column-major matrix described by the following"? Or is order not really even a thing here?

Also, I'm going to get super nit-picky and ask what you think of ASCII art like this:

```
| xx  yx  tx |
| xy  yy  ty |
|  0   0   1 |

Reply:
Can add that!